ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__amd64-nightly-4.18-upgrade-from-stable-4.18.yaml

```diff
@@ -0,0 +1,891 @@
+base_images:
+  ansible:
+    name: "4.18"
+    namespace: ocp
+    tag: ansible
+  assisted-service-scripts:
+    name: ocm-2.11
+    namespace: edge-infrastructure
+    tag: assisted-service-scripts
+  aws-efs-csi-operator-create-efs:
+    name: "4.18"
+    namespace: ocp
+    tag: aws-efs-csi-operator-create-efs
+  cli:
+    name: "4.18"
+    namespace: ocp
+    tag: cli
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  hypershift-operator:
+    name: "4.18"
+    namespace: ocp
+    tag: hypershift-operator
+  openstack-installer:
+    name: "4.18"
+    namespace: ocp
+    tag: openstack-installer
+  tests-private-postupg:
+    name: tests-private
+    namespace: ci
+    tag: "4.18"
+  tests-private-preupg:
+    name: tests-private
+    namespace: ci
+    tag: "4.18"
+  tools:
+    name: "4.18"
+    namespace: ocp
+    tag: tools
+  upi-installer:
+    name: "4.18"
+    namespace: ocp
+    tag: upi-installer
+  verification-tests:
+    name: verification-tests
+    namespace: ci
+    tag: latest
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.18
+releases:
+  latest:
+    prerelease:
```

Comment:
To be consistent with 4.17 z-stream upgrade ci yaml (openshift-openshift-tests-private-release-4.17__amd64-nightly-4.17-upgrade-from-stable-4.17.yaml), shall we update it to the follow setting ?
```
  latest:
    release:
      architecture: amd64
      channel: candidate
      version: "4.18"
```

Reply:
@jianlinliu 4.18 stable builds are still with 4-dev-preview channel and not with 4-stable, so kept it as prerelease. I am not sure whether it will work dev-preview if we keep it as same as 4.17.

Reply:
From openshift-openshift-tests-private-release-4.18__multi-stable-4.18-upgrade-from-stable-4.17.yaml, I can see:
```
  target:
    release:
      architecture: multi
      channel: candidate
      version: "4.18"
```
It already starts to use 4.18 stable. And from the output, I also can see 4.18.0-ec.0:
```
[root@preserve-jialiu ~]# curl --silent --header 'Accept:application/json' https://api.openshift.com/api/upgrades_info/v1/graph?channel=candidate-4.18| jq .|grep 4.18
      "version": "4.18.0-ec.0"
```
Can yo have a try and run a pj-rehearse job to confirm that ?

Reply:
The pj-rehearse job got stuck because of a known issue - `exec /tools/entrypoint: argument list too long`

I tried in my another PR, it works, refer to https://github.com/openshift/release/pull/56899#issuecomment-2359882809.

Reply:
Thanks @jianlinliu